target/linux/qualcommax/ipq807x/base-files/etc/board.d/02_network

```diff
@@ -25,6 +25,9 @@ ipq807x_setup_interfaces()
 	netgear,rax120v2)
 		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4 lan5" "wan"
 		;;
+	netgear,wax620)
```

Comment:
Merge it with `wax218` as they are using the same config.

Reply:
Fixed.